runtime/vara/src/lib.rs

```diff
@@ -959,6 +1147,9 @@ construct_runtime!(
         StakingRewards: pallet_gear_staking_rewards = 106,
         GearVoucher: pallet_gear_voucher = 107,
 
+        // TODO: Remove in stage 3
```

Comment:
Do we need to place some comment that previously validator_set pallet had index 98 and we shouldn't give the index to any other pallet in future?